akka-http-core/src/test/scala/akka/http/impl/model/parser/HttpHeaderSpec.scala

```diff
@@ -722,6 +764,8 @@ class HttpHeaderSpec extends FreeSpec with Matchers {
       try parsedHeader should equal(header)
       catch {
         case e: TestFailedException if parsedHeader.toString == header.toString ⇒
+          println(s"!!!!!!$parsedHeader")
+          println(s"!!!!!!$header")
```

Comment:
No need to include these printlns

Reply:
This was clearly a mistake!
Sorry for that